test_team_tech_bros.py

```diff
@@ -21,6 +21,27 @@ def test_it_identity_card(self):
     def test_it_passport(self):
         """Test IT_PASSPORT functionality"""
 
+        # Valid examples
+        p_num = "JF3349917"
+        invalid_p = "12ABC5678"
+        
+        # Test valid passport with context
+        result = analyze_text(f"My italiano passaporto number is {p_num}", ['IT_PASSPORT'])
+        print(result)
+        self.assertEqual('IT_PASSPORT', result[0].entity_type)
+        self.assertAlmostEqual(0.4, result[0].score, places=2)
+
+        # Test valid passport without context
+        result = analyze_text(f"My number is {p_num}", ['IT_PASSPORT'])
+        print(result)
```

Comment:
Remove Print Statements: The print(result) statements are not needed in unit tests. They can clutter test output and are typically not used in automated testing environments. If you need to debug, consider using logging or assert statements to verify expected conditions instead.